conf/application.conf

```diff
@@ -71,6 +71,10 @@ microservice {
       host = localhost
       port = 9250
     }
+    customer-information {
```

Comment:
Do we need this config entry? I thought customer info was covered under vat-api.